config/engines.yml

```diff
@@ -155,3 +155,9 @@ scss-lint:
   enable_regexps:
   default_ratings_paths:
     - "**.scss"
+requiresafe:
+  image: codeclimate/codeclimate-requiresafe
+  description: Security tool for Node.js dependencies
+  community: true
+  enable_regexps:
+  default_ratings_paths:
```

Comment:
@pbrisbin This tool only looks at one file, and doesn't impact grades yet, so I think we're fine for now.